app/assets/hee/blocks/scaffolding/notifications/phasebanner/_phasebanner.scss

```diff
@@ -1,36 +1,38 @@
 /* NHSUK-HEE-PhaseBanner - swap to HEE-PhaseBanner */
 .nhsuk-hee-phasebanner {
```

Comment:
We named the class `nhsuk-hee-phasebanner` instead of `hee-phasebanner` because it comes from nhsuk prototype? Will it be appropriate to call this as `hee-phasebanner` for consistency?